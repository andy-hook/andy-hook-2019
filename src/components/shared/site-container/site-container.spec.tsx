import React from "react"
import renderer from "react-test-renderer"

import SiteContainer from "./site-container"

describe("<SiteContainer />", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<SiteContainer>Child text</SiteContainer>)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
