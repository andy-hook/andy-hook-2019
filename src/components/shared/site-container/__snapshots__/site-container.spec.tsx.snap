// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SiteContainer /> renders correctly 1`] = `
<div
  className="sc-bdVaJa kxzpCg"
>
  Child text
</div>
`;
